.gitignore

```diff
@@ -46,53 +46,55 @@ MockUtils/Package.resolved
 MockUtils/.build/
 MockUtils/*.xcodeproj/
 
-StitchCoreSDK/Packages/
-StitchCoreSDK/Package.pins
-StitchCoreSDK/Package.resolved
-StitchCoreSDK/.build/
-StitchCoreSDK/*.xcodeproj/
-
-StitchCoreAdminClient/Packages/
-StitchCoreAdminClient/Package.pins
-StitchCoreAdminClient/Package.resolved
-StitchCoreAdminClient/.build/
-StitchCoreAdminClient/*.xcodeproj/
-
-StitchCoreTestUtils/Packages/
-StitchCoreTestUtils/Package.pins
-StitchCoreTestUtils/Package.resolved
-StitchCoreTestUtils/.build/
-StitchCoreTestUtils/*.xcodeproj/
-
-StitchCoreAWSS3Service/Packages/
-StitchCoreAWSS3Service/Package.pins
-StitchCoreAWSS3Service/Package.resolved
-StitchCoreAWSS3Service/.build/
-StitchCoreAWSS3Service/StitchCoreAWSS3Service.xcodeproj/
-
-StitchCoreAWSSESService/Packages/
-StitchCoreAWSSESService/Package.pins
-StitchCoreAWSSESService/Package.resolved
-StitchCoreAWSSESService/.build/
-StitchCoreAWSSESService/StitchCoreAWSSESService.xcodeproj/
-
-StitchCoreHTTPService/Packages/
-StitchCoreHTTPService/Package.pins
-StitchCoreHTTPService/Package.resolved
-StitchCoreHTTPService/.build/
-StitchCoreHTTPService/StitchCoreHTTPService.xcodeproj/
-
-StitchCoreRemoteMongoDBService/Packages/
-StitchCoreRemoteMongoDBService/Package.pins
-StitchCoreRemoteMongoDBService/Package.resolved
-StitchCoreRemoteMongoDBService/.build/
-StitchCoreRemoteMongoDBService/StitchCoreRemoteMongoDBService.xcodeproj/
-
-StitchCoreTwilioService/Packages/
-StitchCoreTwilioService/Package.pins
-StitchCoreTwilioService/Package.resolved
-StitchCoreTwilioService/.build/
-StitchCoreTwilioService/StitchCoreTwilioService.xcodeproj/
+Core/StitchCoreSDK/Packages/
```

Comment:
[q] Do we need to use the fully qualified name for these one they are in subdirs? i.e. can `StitchCoreSDK` become `sdk`?

Reply:
As discussed offline, this directory structure is the best we can do combined with our workspace.